cmd/operator/deploy/operator/10-collector.yaml

```diff
@@ -117,8 +117,10 @@ spec:
           limits:
             memory: 3G
           requests:
-            cpu: 100m
-            memory: 200M
+            # Resource requests were empirically determined to be in this range
+            # when running idle (i.e. no scraping).
+            cpu: 2m
```

Comment:
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#how-pods-with-resource-limits-are-run

says

```
The CPU request typically defines a weighting. If several different containers (cgroups) want to run on a contended system, workloads with larger CPU requests are allocated more CPU time than workloads with small requests.

```

So I am concerned that if we set CPU request for the IDLE phase, the CPU may be taken away by other workloads.

Reply:
See https://github.com/GoogleCloudPlatform/prometheus-engine/pull/394#issuecomment-1347421158